spec/features/user/account/social_profiles/add_remove_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+feature "Add/Remove social profiles" do
+  let(:user) { create(:user, :confirmed) }
+  let(:user_attributes) { user.attributes.slice(:full_name, :email) }
+  let(:oauth) { omniauth_mock(provider, "12345", user_attributes) }
+
+  before do
+    stub_omniauth(provider, oauth)
+    allow(AuthVerificationPolicy).to receive(:verified?)
+
+    login_as user
+    visit edit_user_registration_path
+  end
+
+  context "when provider is Facebook" do
+    let(:provider) { :facebook }
+
+    scenario "user adds social profile" do
+      expect(page).to have_link("Facebook")
+      expect(page).not_to have_content("Successfully authorized via:")
+
+      click_link "Facebook"
+      expect(page).to have_content(I18n.t "devise.omniauth_callbacks.success", kind: "Facebook")
```

Comment:
Uh, if I18n.t won't have this translation, will this test pass?) 
It might expect to see an error on page, and test might pass

Would be better to use just simply text for testing